frontend/src/components/common/Grid/components/DownloadButton/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { StyledOutlineButton } from "src/components/common/Button/common/style";
+import { Intent } from "@blueprintjs/core";
+import { track } from "src/common/analytics";
+import { EVENTS } from "src/common/analytics/events";
+
+interface Props {
+  datasetName: string;
+  onClick: () => void;
+}
+
+export default function DownloadButton({
```

Comment:
Refactored [DownloadButton](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/frontend/src/components/Datasets/components/Grid/common/utils.tsx) to its own component and added tracking for the event "DOWNLOAD_DATA_CLICKED".